README.md

```diff
@@ -65,7 +60,7 @@ No build is required. You may download the source code by doing one of these ste
 
 After downloading the extension to your computer, just load it by following these steps.
 
-- Google Chrome/Microsoft Edge
+- Google Chrome / Microsoft Edge
```

Comment:
Nah, I don't think a space is needed between the slashes.

Reply:
I guess I changed this because `Chrome/Microsoft` looks weird. Wouldn't have changed it if it was `Chrome/Edge`.

Reply:
> I guess I changed this because `Chrome/Microsoft` looks weird. Wouldn't have changed it if it was `Chrome/Edge`.

I see why you did that, but there shouldn't be spaces. A word could be used instead if you really don't like a lack of spaces, though I wouldn't recommend that. I suggest keeping it as it was.

Source: As a native American English speaker, I find no spaces to be the most natural.

Reply:
I'll undo this line change